build.ts

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env
+import {
+  AlphaColor,
+  Color,
+  Labels,
+  variants,
+} from "npm:@catppuccin/palette@0.1.5";
+import { titleCase } from "https://deno.land/x/case@2.2.0/mod.ts";
+import { JSZip } from "https://deno.land/x/jszip@0.11.0/mod.ts";
+import * as uuid from "https://deno.land/std@0.207.0/uuid/mod.ts";
+
+// DON'T CHANGE THIS :OMEGALUL:
```

Comment:
Sorry, meant to change this comment before pushing yesterday.

```suggestion
// Allows for the UUIDs to be "seeded" and therefore reproducible
// DO NOT CHANGE
```